src/DFC_BusinessOntology.owl

```diff
@@ -8914,6 +10154,28 @@
         <IRI>#proposes</IRI>
         <Literal xml:lang="en">proposes</Literal>
     </AnnotationAssertion>
+    <AnnotationAssertion>
```

Comment:
This is really confusing. Can we find a way to allocate these to the specific Domain they're relevant to ?